### ISSUE TITLE
apache_enum_users fails for cgiwrap

### ISSUE THREAD

**Issue by sullo** [sullo/nikto#15]
The code for apache_enum_users always returns if it doesn't succeed with a ~user test.

As we could do with adding entropysearch; it'll probably be quicker to rewrite the logic of this plugin.

---

Basically, the logic for the apache_enum_users plugin goes:

Test to see whether ~user works
If not leave
Test to see whether cgiwrap is going be used then enum with cgiwrap
Otherwise enum with ~user
So basically it won't enum with cgiwrap unless you can enum with ~user.
